### ISSUE TITLE
WCAG2Mobile: Draft feedback - 1.4.2 Audio Control

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#651]
(Mary Jo) 

- Disagree with using "view" in the MATF document. There is no need to change the application of the SC to a view. 
- Phil's recorded issue, https://github.com/w3c/wcag2ict/issues/638 solves this problem

(Chris L) 

- +1 to Mary Jo's point.

**Comment by pday1** [comment]
Closed - insufficient time or resource to review each WCAG2Mobile clause
